charts/prefect-worker/README.md

```diff
@@ -298,6 +298,7 @@ Please note that configuring the template via `baseJobTemplate.existingConfigMap
 | worker.initContainer.containerSecurityContext.readOnlyRootFilesystem | bool | `true` | set init containers' security context readOnlyRootFilesystem |
 | worker.initContainer.containerSecurityContext.runAsNonRoot | bool | `true` | set init containers' security context runAsNonRoot |
 | worker.initContainer.containerSecurityContext.runAsUser | int | `1001` | set init containers' security context runAsUser |
+| worker.initContainer.extraContainers | list | `[]` | additional sidecar containers |
```

Comment:
Looks like this was accidentally missed in https://github.com/PrefectHQ/prefect-helm/pull/403